trojsten/notifications/utils.py

```diff
@@ -0,0 +1,23 @@
+from .models import Notification, UnsubscribedChannel
+
+
+def notify(users, channel, message, url):
+    """
+    Notifies all users in `users` that are not unsubscribed.
+
+    Arguments:
+    users -- iterable containing instances of trojsten.people.models.User
+    channel -- notification channel name
+    message -- display message for user
+    url -- URL to open when read
+    """
+
+    unsubscribed = UnsubscribedChannel.objects.filter(channel=channel).values_list(
+        "user__pk", flat=True
+    )
+
+    for user in users:
+        if user.pk in unsubscribed:
+            continue
+
+        Notification.objects.create(user=user, channel=channel, message=message, url=url)
```

Comment:
Toto vie vyrobit potencialne vela notificatiion objektov, toz aj vela queries naraz.
Potencialne https://docs.djangoproject.com/en/2.2/ref/models/querysets/#bulk-create by mohlo trochu pomoct, ale eventualne tento call nechceme robit realtime. Najlepsie by bolo pouzit celery a mat to ako background task.
Myslim ze s momentalnym mnozstvom userov ktorym sa maju posielat notifikacie by to nemal byt problem, ale treba vyrobit issue, ze to chceme prekopat na celery, a dat sem TODO.